main.scss

```
@@ -0,0 +1,58 @@
+@charset "utf-8";
+
+body {
+  margin:           0;
+  padding:          0;
+  background-color: #cccccc;
+  color:            #333333;
+  font-size:        15px;
+  line-height:      2;
+  position:         relative;
+}
+
+ul { margin: 0; padding: 0; }
+
+a { color: #3583aa; text-decoration: none; }
+
+a:hover { text-decoration: underline; }
```

こういったデフォルトのcss 指定はコメントアウトで
```suggestion

// reset css
body {
  margin:           0;
  padding:          0;
  background-color: #cccccc;
  color:            #333333;
  font-size:        15px;
  line-height:      2;
  position:         relative;
}

ul { margin: 0; padding: 0; }

a { color: #3583aa; text-decoration: none; }
a:hover { text-decoration: underline; } 
```
のように書いておくとわかりやすいと思います

aとa:hoverは密接な関係なのでこういった時は一緒にくっつけてあげた方がビジュアル的には良さそうです！